operator/deploy/crd.yaml

```diff
@@ -30,11 +30,335 @@ spec:
     listKind: KialiList
     plural: kialis
     singular: kiali
+  preserveUnknownFields: false
   scope: Namespaced
-  subresources:
-    status: {}
-  version: v1alpha1
   versions:
-  - name: v1alpha1
-    served: true
-    storage: true
+    - name: v1alpha1
```

Comment:
We also want to finally bump up our schema version officially to "v1". I think with this change of adding a full schema, we can say we are no longer in alpha (v1alpha1).

This means we have to investigate what we need to do to properly upgrade CRD versions, particularly within OLM.

Reply:
When we bump up the version to v1, all kiali CR files in operator/deploy/kiali need to be moved up to the new version. For example:

https://github.com/kiali/kiali/blob/v1.9/operator/deploy/kiali/kiali_cr.yaml#L1

should be using `apiVersion: kiali.io/v1`

Reply:
Also, the operator needs to be told to watch for the new version:

https://github.com/kiali/kiali/blob/v1.9/operator/watches.yaml#L2

I think we should still be supporting the v1alpha1 - so we'll need to add the new version to the watches.yaml rather than just replacing the current v1alpha1 version.